diff --git a/cmake/CommonModules/TinyCommon.cmake b/cmake/CommonModules/TinyCommon.cmake
index 39901b6..f298451 100644
--- a/cmake/CommonModules/TinyCommon.cmake
+++ b/cmake/CommonModules/TinyCommon.cmake
@@ -137,9 +137,6 @@ ${TINY_UNPARSED_ARGUMENTS}")
             $<$<CONFIG:Debug,RelWithDebInfo>:
                 LINKER:--dynamicbase,--high-entropy-va,--nxcompat
                 LINKER:--default-image-base-high>
-            # Use faster linker ( GNU ld linker doesn't work with the Clang;
-            # for both GCC and Clang )
-            -fuse-ld=lld
         )
     endif()
 
